Sources/Commands/SwiftTestTool.swift

```
@@ -148,16 +149,17 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 }
             }
 
-            exit(ranSuccessfully ? 0 : 1)
+            return ranSuccessfully ? .success : .failure(1)
 
         case .runParallel:
             let testPath = try buildTestsIfNeeded(options)
             let testSuites = try getTestSuites(path: testPath)
             let tests = testSuites.filteredTests(specifier: options.testCaseSpecifier)
             let runner = ParallelTestRunner(testPath: testPath, processSet: processSet)
             try runner.run(tests)
-            exit(runner.ranSuccesfully ? 0 : 1)
```

We need to return the failure here, right?